src/client/GameWindow.cpp

```diff
@@ -80,9 +80,52 @@ GameWindow::GameWindow() {
                 switch (event.type)
                 {
                 case sf::Event::MouseButtonPressed:
+            
+                    clickStartingPoint = {  sf::Mouse::getPosition(clientGameWindow).x, 
+                                            sf::Mouse::getPosition(clientGameWindow).y};
+
+                    if (!mooveMode) {
+
+                        int minimumDistance = WINDOW_LENGTH;
+                        std::array<int, 2> hexagonOnClick = {0, 0};
+                        std::array<int, 2> firstHexagon = {WINDOW_LENGTH, WINDOW_WIDTH};
```

Comment:
I don't think "firstHexagon" is self explanatory name as this variable represents screen dimensions.

Reply:
Done